Docs/variational/Vmc.md

```diff
@@ -5,19 +5,19 @@ Variational Monte Carlo schemes to learn the ground state using stochastic recon
+|target                   |str = 'energy'           |The chosen target to minimize. Possible choices are `energy`, and `variance`.                                                                                                                                                                                                                                                                                                                                                                                 |
+|method                   |str = 'Sr'               |The chosen method to learn the parameters of the wave-function. Possible choices are `Gd` (Regular Gradient descent), and `Sr` (Stochastic reconfiguration a.k.a. natural gradient).                                                                                                                                                                                                                                                                          |
+|diag_shift               |float = 0.01             |The regularization parameter in stochastic reconfiguration. The default is 0.01.                                                                                                                                                                                                                                                                                                                                                                              |
+|use_iterative            |bool = False             |Whether to use the iterative solver in the Sr method (this is extremely useful when the number of parameters to optimize is very large). The default is false.                                                                                                                                                                                                                                                                                                |
+|sr_lsq_solver            |str = 'BDCSVD'           |The solver used to solve the least-squares equation in the SR update. Only used if `method == "SR" and not use_iterative`. Available options are "BDCSVD", "ColPivHouseholder", "LDLT", and "LLT". See the [Eigen documentation](https://eigen.tuxfamily.org/dox/group__TutorialLinearAlgebra.html) for a description of the available solvers. Also note that LLT and LDLT only work for positive definite or positive semidefinite S matrices, respectively.|
```

Comment:
maybe worth setting the default to cholesky?

Reply:
Yes, I'd set `LLT` the default value, it is much faster and it should also be relatively safe if there is a finite `diag_shift`...